webpack/react_app/components/RegistrationExtension/__tests__/RexInterface.test.js

```diff
@@ -1,9 +1,19 @@
-import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
 import RexInterface from '../RexInterface';
 
 const fixtures = {
   renders: { isLoading: false, onChange: () => {} },
 };
 
-describe('RexInterface', () =>
-  testComponentSnapshotsWithFixtures(RexInterface, fixtures));
+describe('RexInterface', () => {
+  it('should render label with help icon and popover instructions', () => {
```

Comment:
Not sure why the github actions are failing, but I ran it locally and it fails with:
```
 ../foreman_remote_execution/webpack/react_app/components/RegistrationExtension/__tests__/RexInterface.test.js (234 MB heap size)
  RexInterface
    ✕ should render label with help icon and popover instructions (40 ms)

  ● RexInterface › should render label with help icon and popover instructions

    TestingLibraryElementError: Unable to find an element with the text: Identifier of the Host interface for Remote execution. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
```